test/functional/legacy/writefile_spec.lua

```diff
@@ -17,13 +19,16 @@ describe('writefile', function()
     execute('bwipeout!')
     execute('$put =readfile(f)')
     execute('1 delete _')
+    execute('w! test.out')
+    execute('call delete(f)')
+    execute('qa!')
```

Comment:
Of these lines, only the second is needed. `test.out` is only for the vim test infrastructure, we have our own that works differently, so neither `test.in` nor `test.out` is needed (note that this is an "old style" test, which was converted to lua here... there are "new style" tests in vim which we take over verbatim (mostly).). You don't need to quit also, that's done by the test suite (I guess it will bring trouble, too).